app/src/main/java/org/mozilla/fenix/components/toolbar/TabCounter.kt

```diff
@@ -48,29 +43,15 @@ open class TabCounter @JvmOverloads constructor(
     }
 
     fun setCountWithAnimation(count: Int) {
-        updateContentDescription(count)
-
-        // Don't animate from initial state.
-        if (this.count == 0) {
-            setCount(count)
-            return
-        }
+        setCount(count)
```

Comment:
Since we're setting the count(so implicitly INTERNAL_COUNT) here, the following `when` case `INTERNAL_COUNT == count` will always be true, so there will never be an animation

Reply:
You're right! I'm going to remove it.